cucoslib/dispatcher/flows/githubManifestMetadataFlow.yml

```diff
@@ -0,0 +1,30 @@
+---
+  flow-definitions:
+    - name: 'githubManifestMetadataFlow'
+      node_args_from_first: true
+      queue: '{DEPLOYMENT_PREFIX}_githubManifestMetadataFlow_v0'
+      edges:
+        - from:
+          to: 'GitHubManifestMetadataInit'
+        - from: 'GitHubManifestMetadataInit'
+          to: 'metadata'
+        - from: 'metadata'
+          to: 'dependency_snapshot'
+        - from: 'dependency_snapshot'
+          to: 'bayesianFlow'
+          foreach:
+            import: 'cucoslib.dispatcher.foreach'
+            function: 'iter_dependencies_analysis'
+            propagate_result: true
+          condition:
```

Comment:
What confused me is the fact that there are no analyses in this flow. But given the fact we want to reuse `iter_dependencies_analysis` that decrements `recursive_limit`, so we might do it this way (zero means no analyses at all).